Trading/Mode/arbitrage_trading_mode/arbitrage_trading.py

```diff
@@ -47,13 +47,15 @@ def get_current_state(self) -> (str, float):
                self.producers[0].final_eval if self.producers[0].final_eval else "N/A"
 
     async def create_producers(self) -> list:
+        default_producers = await super().create_producers()
```

Comment:
Why do we need to call `await super().create_producers()` in every trading mode now ?

Reply:
I moved it to the trading_modes_updates branch and changed it to self.create_default_consumers() and self.create_default_producers()